src/worldstate/include/routines/Gate/AlignWithGate.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Created By: Joel Ahn
+ * Created On: March 5th, 2018
+ * Description: World State FSM node. Checks to see if
+ *              robot still needs to align with gate.
+ */
+
+#ifndef PROJECT_ALIGNWITHGATE_H
+#define PROJECT_ALIGNWITHGATE_H
+
+#include "../State.h"
```

Comment:
We should replace this with `#include "State.h"`, which basically just makes it the responsibility of the build system (cmake) to find this header file for us (rather then us specifying a relative path to it), which makes potential changes and file movements much easier and cleaner in the future. Just noting it here, but there are a few other places that we should also make this change.

Reply:
Changed it to #include "routines/State.h" for the three State subclass header files

Reply:
If your Cmake is done correctly it should be just `#include "State.h` as Gareth mentioned

Reply:
Ah you're right. Overlooked the include_directories line

Reply:
Personally I don't object to `#include "routines/State.h"`, the objection here was to relative file paths. Regardless, this looks good.